Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -622,8 +647,23 @@ void NetPlayDialog::UpdateGUI()
       m_players_list->selectRow(i);
   }
 
-  // Update Room ID / IP label
-  if (m_use_traversal && m_room_box->currentIndex() == 0)
+  if (m_old_player_count != m_player_count)
+  {
+    UpdateDiscordPresence();
+    m_old_player_count = m_player_count;
+  }
+
+  if (!server)
+    return;
+
+  const bool is_local_ip_selected = (m_room_box->currentIndex() - m_use_traversal) == 1;
```

Comment:
Yeah I know this technically works, but it looks weird. I think it's more clear using a ternary like was formerly on [line 654](https://github.com/dolphin-emu/dolphin/pull/10493/files#diff-266b4d0c1c52a05bdb8aa5e08ceb94ed5de284cb882eb27192893856a26400deL654).

Reply:
Fixed.